db/migrate/20240208134335_add_payment_url_to_made_live_forms.rb

```diff
@@ -0,0 +1,22 @@
+class AddPaymentUrlToMadeLiveForms < ActiveRecord::Migration[7.1]
+  class MadeLiveForm < ApplicationRecord
+    self.table_name = "made_live_forms"
+  end
```

Comment:
I'm happy to remove it, would it be more appropriate to name the inline model stub as `MadeLiveFormStub`, so that it's disconnected from the actual model properly?

Reply:
> I'm happy to remove it, would it be more appropriate to name the inline model stub as MadeLiveFormStub, so that it's disconnected from the actual model properly?

I'm not sure 🤔  I mean in this case you're shadowing the existing model, so there's probably no concern about the actual model bleeding through?

Searching for examples in alphagov shows that most other projects don't bother to add the `Stub` suffix, so maybe that's a good guide?

https://github.com/search?q=org%3Aalphagov+path%3A%2F%5Edb%5C%2Fmigrate%5C%2F%2F+ApplicationRecord&type=code

@chao-xian might also have thoughts.

Reply:
Happy for it to be removed too. I did note it but wasn't too fussed at the time. Thanks for mentioning Laurence!

Reply:
It's gone!

Reply:
Sorry I didn't mean for the stub entirely being removed, just the `self.table_name`.

This isn't about solving models bleeding, but because migrations are historical. We'll never remove or rename a class named Forms, but MadeLiveForms is a possibility. 

Having the `class MadeLiveForms < ActiveRecord end;` in a migration stops a migration error.